kubernetes/resource_kubernetes_pod_test.go

```diff
@@ -722,6 +722,78 @@ func TestAccKubernetesPod_config_container_startup_probe(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesPod_termination_message_policy_default(t *testing.T) {
+	var confPod api.Pod
+
+	podName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	imageName := "nginx:1.7.9"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckKubernetesPodDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesTerminationMessagePolicyDefault(podName, imageName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesPodExists("kubernetes_pod.test", &confPod),
+					resource.TestCheckResourceAttr("kubernetes_pod.test", "spec.0.container.0.termination_message_policy", "File"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccKubernetesPod_termination_message_policy_override_as_file(t *testing.T) {
+	var confPod api.Pod
+
+	podName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	imageName := "nginx:1.7.9"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckKubernetesPodDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesTerminationMessagePolicyWithFile(podName, imageName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesPodExists("kubernetes_pod.test", &confPod),
+					resource.TestCheckResourceAttr("kubernetes_pod.test", "spec.0.container.0.termination_message_policy", "File"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err(t *testing.T) {
+	var confPod api.Pod
+
+	podName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	imageName := "nginx:1.7.9"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckKubernetesPodDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesTerminationMessagePolicyWithFallBackToLogsOnErr(podName, imageName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesPodExists("kubernetes_pod.test", &confPod),
+					resource.TestCheckResourceAttr("kubernetes_pod.test", "spec.0.container.0.termination_message_policy", "FallBackToLogsOnError"),
```

Comment:
This test fails for me on GKE with version 1.16 with this error. Re-testing with 1.17
```
=== RUN   TestAccKubernetesPod_termination_message_policy_default
--- PASS: TestAccKubernetesPod_termination_message_policy_default (18.44s)
=== RUN   TestAccKubernetesPod_termination_message_policy_override_as_file
--- PASS: TestAccKubernetesPod_termination_message_policy_override_as_file (10.44s)
=== RUN   TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err
    TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err: testing.go:654: Step 0 error: Check failed: 1 error occurred:
        	* Check 2/2 error: kubernetes_pod.test: Attribute 'spec.0.container.0.termination_message_policy' expected "FallBackToLogsOnError", got "FallbackToLogsOnError"


--- FAIL: TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err (10.04s)
FAIL
```

Reply:
Seems to be a minor typo - FallBack vs Fallback

Reply:
Same failure, but the other tests also pass on 1.17
```
❯ TESTARGS="-run '^TestAccKubernetesPod_termination_message_policy'" make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test "./kubernetes" -v -run '^TestAccKubernetesPod_termination_message_policy' -timeout 120m
=== RUN   TestAccKubernetesPod_termination_message_policy_default
--- PASS: TestAccKubernetesPod_termination_message_policy_default (11.34s)
=== RUN   TestAccKubernetesPod_termination_message_policy_override_as_file
--- PASS: TestAccKubernetesPod_termination_message_policy_override_as_file (15.33s)
=== RUN   TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err
    TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err: testing.go:654: Step 0 error: Check failed: 1 error occurred:
        	* Check 2/2 error: kubernetes_pod.test: Attribute 'spec.0.container.0.termination_message_policy' expected "FallBackToLogsOnError", got "FallbackToLogsOnError"


--- FAIL: TestAccKubernetesPod_termination_message_policy_override_as_fallback_to_logs_on_err (22.92s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-kubernetes/kubernetes	51.639s
FAIL
make: *** [testacc] Error 1
```